packages/common/container-definitions/src/index.ts

```diff
@@ -70,3 +62,28 @@ export {
 	IRuntime,
 	IRuntimeFactory,
 } from "./runtime";
+
+/**
```

Comment:
I don't believe the doc propagation will work correctly this way. I believe you have to do something like:

```typescript
export {
/**
 * @deprecated IErrorBase is being deprecated as a public export is moving to "core-interfaces".
 */
IErrorBase,
...
} from "...";
```

Reply:
That way the deprecated comment tag is associated with the individual exported member, rather than the export statement.